pkg/cli/internal/initialize/conduit.yml.example

```diff
@@ -33,3 +33,12 @@ processors:
 # An exporter is defined to do something with the data.
 exporter:
 %s
+
+# Enable telemetry for conduit
+telemetry:
+  enabled: true
+  # These configs define default values for initializing the telemetry server.
+  uri: ""
+  index: ""
+  username: ""
+  password: ""
```

Comment:
```suggestion

  # By default the following fields will be configured to send data to Algorand.
  # To store your own telemetry events, they can be overridden.
  # uri: ""
  # index: ""
  # username: ""
  # password: ""
```

Reply:
Updated in [96a7c9c](https://github.com/algorand/conduit/pull/74/commits/96a7c9cfa4b3858828e10fa74910d2492fc53654)